src/app/testing/synapse-mocks.ts

```diff
@@ -4,5 +4,5 @@ export const synapseWikiMock: SynapseWiki = {
+    'Agora hosts evidence for whether or not genes are associated with Alzheimer\u2019s disease (AD). Agora also contains a list of over 600 nascent drug targets for AD that were nominated by AD researchers. The list of nominated targets was contributed by researchers from the <a href="https://www.nia.nih.gov/research/amp-ad" target="_blank" rel="noopener noreferrer">National Institute on Aging\'s Accelerating Medicines Partnership in Alzheimer\'s Disease (AMP-AD) consortium</a> as well as other <a href="https://agora.adknowledgeportal.org/teams" target="_blank" rel="noopener noreferrer">research teams</a>.  Other evidence presented in Agora was either generated by AMP-AD research teams or is aggregated from publicly available data sources.\n&nbsp;\nAdditional information about Agora is available in the <a href="https://help.adknowledgeportal.org/apd/Agora-Resources.2646671361.html" target="_blank" rel="noopener noreferrer">Agora Resources documentation</a>.  \n&nbsp;\nWe encourage you to use the data and insights presented in Agora along with the following acknowledgement statement: \n&nbsp;\n<blockquote>The results published here are in whole or in part based on data obtained from Agora, a platform initially developed by the NIA-funded AMP-AD consortium that shares evidence in support of AD target discovery.</blockquote>\n&nbsp;\nAgora is funded by the <a href="https://www.nia.nih.gov/" target="_blank" rel="noopener noreferrer">National Institute on Aging</a>. It is developed and maintained by <a href="https://sagebionetworks.org/" target="_blank" rel="noopener noreferrer">Sage Bionetworks</a>. If you have questions, suggestions, or feedback about Agora, please contact us at: **agora@sagebionetworks.org**.\n\n\n',
```

Comment:
The only change here is `https://agora.ampadportal.org/teams-contributing` -> `https://agora.adknowledgeportal.org/teams`